modules/Walking_module/CMakeLists.txt

```diff
@@ -69,14 +69,16 @@ include_directories(
   ${qpOASES_INCLUDEDIR}
 )
 
-# Thrift for the RPC commands
-yarp_add_idl(THRIFT "${CMAKE_CURRENT_SOURCE_DIR}/thrifts/WalkingCommands.thrift")
-add_library(icubWalking-service STATIC ${THRIFT})
-target_include_directories(icubWalking-service SYSTEM PUBLIC ${YARP_INCLUDE_DIRS})
-target_link_libraries(icubWalking-service YARP::YARP_init YARP::YARP_OS)
+set(${EXE_TARGET_NAME}_THRIFT_HDR
+  thrifts/WalkingCommands.thrift
+  )
+
+# Application target calls
+yarp_add_idl(${EXE_TARGET_NAME}_THRIFT_SRC ${${EXE_TARGET_NAME}_THRIFT_HDR})
```

Comment:
I am just not super convinced about the name ``${EXE_TARGET_NAME}_THRIFT_SRC``. Those are actually autogenerated files, not source files you wrote. I would go for ``${EXE_TARGET_NAME}_THRIFT_GEN_FILES``.

Reply:
Done in 81491bb